### ISSUE TITLE
bind parameter error

### ISSUE THREAD

**Issue by hellower** [tarantool/pg#14]
```
localhost:3301> box.info.version
---
- 1.7.5-238-g39276fe
...

localhost:3301> local pg = require('pg')
---
...

localhost:3301> local conn = pg.connect({host = '127.0.0.1', user = 'mig', pass = 'mig', db = 'perf', port=5432 })
---
...

localhost:3301> conn:execute('select 100')
---
- - - ?column?: 100
- true
...

localhost:3301> conn:execute('select ? as t', 100)
---
- error: "ERROR:  syntax error at or near \"as\"\nLINE 1: select ? as t\n                 ^\n"
...

localhost:3301> 
```

**Comment by nemat-rakhmatov** [comment]
It seems documentation of pg module incorrectly states ? as a parameter for postgresql statement, it should be $1, $2, etc. as required by libpq

**Comment by Totktonada** [comment]
The README was fixed in 2a9b9e5737762578dfd9067c40f4ad9b1c853c9e and 737f0f6d93f82a9ea9839f2e024f5b8e3181e927.
